core/templates/dev/head/footer.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>

```

Comment:
Don't need this, and don't need 'html' tag below. This is an HTML fragment which will be inserted in the page.